airbyte-webapp/src/pages/SettingsPage/pages/NotificationPage/components/WebHookForm.tsx

```diff
@@ -155,13 +155,7 @@ export const WebHookForm: React.FC<WebHookFormProps> = ({ webhook }) => {
                     </a>
                   </li>
                 </ul>
-                <img
-                  className={styles.webhookGuideImg}
-                  alt={formatMessage({
-                    id: "settings.notificationGuide.button",
```

Comment:
:information_source: This `id` was still used in another place, so no removal from `en.json`